src/index.cpp

```diff
@@ -511,7 +512,7 @@ namespace diskann {
       if (file_exists(universal_label_file)) {
         std::ifstream universal_label_reader(universal_label_file);
         universal_label_reader >> _universal_label;
-        _use_universal_label = true;
+        _use_universal_label = true; // to check
```

Comment:
What is `// to check` referring to here?

Reply:
This was to try debugging something with the disk build. Not relevant to in-memory

Reply:
I'd recommend either being more explicit in the comment or removing it if it's no longer relevant

Reply:
done